lib/routes/alicesoft/infomation.ts

```diff
@@ -0,0 +1,91 @@
+import { Route } from '@/types';
+import got from '@/utils/got';
+import cache from '@/utils/cache';
+import { load } from 'cheerio';
+
+const baseUrl = 'https://www.alicesoft.com';
+
+export const route: Route = {
+    url: 'www.alicesoft.com/information',
+    path: '/information/:category?/:game?',
+    categories: ['game'],
+    example: '/alicesoft/information/game/cat377',
+    parameters: {
+        category: 'Category in the URL, which can be accessed under カテゴリ一覧 on the website.',
+        game: 'Game-specific subcategory in the URL, which can be accessed under カテゴリ一覧 on the website. In this case, the category value should be `game`.',
+    },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['www.alicesoft.com/information', 'www.alicesoft.com/information/:category', 'www.alicesoft.com/information/:category/:game'],
+            target: '/information/:category/:game',
+        },
+    ],
+    name: 'ニュース',
+    maintainers: ['keocheung'],
+    handler,
+};
+
+async function handler(ctx) {
+    const { category, game } = ctx.req.param();
+    const limit = ctx.req.query('limit') ? Number.parseInt(ctx.req.query('limit'), 10) : 10;
+
+    let url = `${baseUrl}/information`;
+    if (category) {
+        url += `/${category}`;
+        if (game) {
+            url += `/${game}`;
+        }
+    }
+
+    const response = await got(url);
+    const $ = load(response.data);
+
+    let items = $('div.cont-main li')
+        .slice(0, limit)
+        .toArray()
+        .map((item) => {
+            item = $(item);
+            return {
+                title: item.find('p.txt').text(),
+                link: item.find('a').attr('href'),
+                pubDate: new Date(item.find('time').attr('datetime')),
+            };
+        });
+
+    items = await Promise.all(
+        items.map((item) => {
+            if (!item.link.startsWith(`${baseUrl}/information/`)) {
+                return item;
+            }
+            return cache.tryGet(item.link, async () => {
+                const detailResponse = await got({
+                    method: 'get',
+                    url: item.link,
+                });
```

Comment:
Please stick with either https://github.com/DIYgod/RSSHub/blob/f90206a165345bb0dac2adc999cfaa4aeefe18ec/lib/routes/alicesoft/infomation.ts#L48 or https://github.com/DIYgod/RSSHub/blob/f90206a165345bb0dac2adc999cfaa4aeefe18ec/lib/routes/alicesoft/infomation.ts#L69-L72 but [not both](https://docs.rsshub.app/joinus/advanced/script-standard#general-guidelines).